docs/docs/guides/exporting-logs.md

```diff
@@ -404,12 +404,112 @@ kurtosis-clusters:
             codec: "json"
 ```
 
+### Configuring Log Filters
+
+Kurtosis allows you to configure filters that get passed through to the logs collector. These filters can be used to modify or filter logs before they are sent to the configured sinks from export logs.
+
+The filters are configured in the `logs-collector` section of your Kurtosis config using version 6. Each filter has a `name`, `match` pattern, and parameters from the corresponding Fluentbit [filter][fluentbit-filters] used.
+
+Here are some examples of common filter configurations that might be useful in Kurtosis.
+
+#### Basic Grep Filter
```

Comment:
Nice job with examples! Might be helpful to link the title to https://docs.fluentbit.io/manual/pipeline/filters/grep just so people know exactly where this comes from